users/static/site_notification/notification.js

```diff
@@ -0,0 +1,55 @@
+let get_notification = "";
+let seen_notification = "";
+
+
+function build_table() {
+	const id = "#notification-table";
+
+	$.get(get_notification, {}, function(data, status) {
+		
+		$(id).empty();
+
+		let length = data.length;
+		if (length === 0) {
+			length = "";
+		}
+
+		$("#notification-badge").text(length);  // change the icon to the amount of notification
+
+		data.forEach((element, index) => {
```

Comment:
can you please add a message when there no new notification, indicating it.

Reply:
done. thanks 😄